tests/Unit/Product/BusinessUnitTest.php

```diff
@@ -108,6 +111,49 @@ public function testItCreatesABusinessUnitWithMandatoryAndOptionalParameters():
         $this->assertEquals($businessUnit->getSaleEndDateString(), $this->specialToDate->format('Y-m-d H:i:s'));
     }
 
+    public function testItCreatesAProductFromAnXml(): void
+    {
+        $this->specialFromDate = DateTimeImmutable::createFromFormat(DATE_ATOM, '2021-5-10T14:54:23+00:00');
+        $this->specialToDate = DateTimeImmutable::createFromFormat(DATE_ATOM, '2021-5-20T14:54:23+00:00');
+        $xml = sprintf(
+            '<BusinessUnit>
+            <BusinessUnit>%s</BusinessUnit>
+            <OperatorCode>%s</OperatorCode>
+            <Price>%f</Price>
+            <SpecialPrice>%f</SpecialPrice>
+            <SpecialFromDate>%s</SpecialFromDate>
+            <SpecialToDate>%s</SpecialToDate>
+            <Stock>%d</Stock>
+            <Status>%s</Status>
+            <IsPublished>%d</IsPublished>
+          </BusinessUnit>',
```

Comment:
could you put this into the `tests/_schemas` folder?  
You can use de `getSchema` method (LinioTestCase.php)

Reply:
Done, could you please check it now. Thanks!